src/ec2/commands.ts

```diff
@@ -30,3 +32,14 @@ export async function tryConnect(selection?: Ec2Selection): Promise<void> {
     const ec2Client = new Ec2ConnectionManager(selection.region)
     await ec2Client.attemptEc2Connection(selection)
 }
+
+export async function copyInstanceId(instanceId: string): Promise<void> {
+    try {
+        if (!instanceId) {
+            throw new ToolkitError(`Attempting to copy undefined instanceId.`)
+        }
+        await copyToClipboard(instanceId)
+    } catch (e) {
+        throw new ToolkitError('Failed to copy instanceId', { code: 'InvalidResource', cause: e as Error })
```

Comment:
This shouldn't really be possible for copy-to-clipboard, but if it happens, the global exception handler is good enough. 

The try/catch in copyArn.ts is likely old/legacy and could be cleaned up: https://github.com/aws/aws-toolkit-vscode/blob/926fca0d358bd36c473febf182e9c30e4a2af198/src/awsexplorer/commands/copyArn.ts#L28-L30